src/main/java/com/bitso/BitsoBalance.java

```diff
@@ -9,15 +9,15 @@
 import com.bitso.helpers.Helpers;
 
 public class BitsoBalance {
-    public BigDecimal mxnTotal;
-    public BigDecimal ethTotal;
-    public BigDecimal btcTotal;
-    public BigDecimal mxnLocked;
-    public BigDecimal ethLocked;
-    public BigDecimal btcLocked;
-    public BigDecimal mxnAvailable;
-    public BigDecimal ethAvailable;
-    public BigDecimal btcAvailable;
+    protected BigDecimal mxnTotal;
+    protected BigDecimal ethTotal;
+    protected BigDecimal btcTotal;
+    protected BigDecimal mxnLocked;
+    protected BigDecimal ethLocked;
+    protected BigDecimal btcLocked;
+    protected BigDecimal mxnAvailable;
+    protected BigDecimal ethAvailable;
+    protected BigDecimal btcAvailable;
```

Comment:
I wonder if there's a nicer way of doing this. For example, through the use of enums. Then your public getters can just pass in the enum and return the relevant BigDecimal.